plugins/rt.js

```diff
@@ -1572,6 +1576,10 @@
 				if (edata.responseBodyNotUsed) {
 					BOOMR.addVar("fetch.bnu", 1, true);
 				}
+
+				if (edata.responseUrl) {
+					BOOMR.addVar("ru", edata.responseUrl, true);
```

Comment:
Make sure to run it through `BOOMR.utils.cleanupURL(url, edata.responseUrl)` so the query string can be stripped if needed.